src/main/java/kata/makers/Checkout.java

```diff
@@ -0,0 +1,18 @@
+package kata.makers;
+
+import java.util.Arrays;
+import kata.makers.exception.InvalidPricesException;
+
+public class Checkout {
+
+  public int getTotal(String prices) throws InvalidPricesException {
+    if (prices.length() == 1 || prices.length() == 2) {
```

Comment:
I think `prices.length() <= 2` can do the same thing!

but sorry that's on me, I didn't mention how should handle prices has length = 0 (i.e.: empty string)